scripts/find-low-dpi-images.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+# Created by Sam Gleske (@samrocketman on GitHub)
+# MIT licensed
+# Sat Aug 27 13:35:25 EDT 2022
+# Pop!_OS 18.04 LTS
+# Linux 5.4.0-113-generic x86_64
+# GNU bash, version 4.4.20(1)-release (x86_64-pc-linux-gnu)
+# find (GNU findutils) 4.7.0-git
+# xargs (GNU findutils) 4.7.0-git
+# git version 2.17.1
+#
+# DESCRIPTION
+#     Find images which exist in endless-sky-high-dpi but are missing from
+#     endless-sky repository.
+#
+#     This script will exit non-zero if missing images are found.
+#
+# USAGE
+#     git clone https://github.com/endless-sky/endless-sky-high-dpi
+#     cd endless-sky-high-dpi/
+#     ../find-low-dpi-images.sh
+
+set -exo pipefail
+
+trap '[ ! -d "${TMP_DIR:-}" ] || rm -rf "${TMP_DIR}"' EXIT
+TMP_DIR="$(mktemp -d)"
+export TMP_DIR
+
+
+pushd "${TMP_DIR}" &> /dev/null
+git clone --depth=1 https://github.com/endless-sky/endless-sky.git
```

Comment:
Why do I need the script to clone ES? I have it cloned already locally. This script should take a path to the root ES directory (you can use the presence of `credits.txt` and `images/` to verify).

Reply:
This is for an action not your local clone.  It can be updated later to work from local
if you want.  i.e. take an arg.

Reply:
Why should we add a script that unnecessarily downloads the ES repo when only the CI environment needs that bit of functionality?

Reply:
Then don't.  You want to have something that does it different you're welcome to provide it.  It was pointed out on Discord that a feature like this was desirable.  It's typical for a CI environment to be able to clone what it's checking against.  I defer to you.

Definitively close this if you don't want it.  I've made the changes I'm going to make for now and anything extra can come later or you can just full reject it.  Up to you.

Reply:
Perhaps it would make more sense for me to explain why I think your requests in this instance didn't make sense to me.

* The majority of those contributing to this repository are on Windows.
* Of the minority who may be on Mac; a small percentage would be technical to fire up a shell and run this script.

This script is designed for CI and intended to be run within CI.